pkg/downloader/ipfs/downloader_test.go

```diff
@@ -130,9 +130,13 @@ func TestDownloadFromPrivateSwarm(t *testing.T) {
 	config.Set(cfg)
 
 	t.Run("download failure without swarm key", func(t *testing.T) {
```

Comment:
:thinking: All we are really doing here is validating we configured IPFS correctly and validating it's behavior with a swarm key configured. Would asserting the IPFS daemon is configured as expected be sufficient, rather than attempting to download a file we can't actually download?

Reply:
I think it's probably okay that we're at least testing that timeouts are honoured if we specify them in the context.